src/components/registration/registrationForm/registrationForm.tsx

```diff
@@ -58,19 +60,45 @@ const validationSchema = Yup.object().shape({
 });
 
 export const RegistrationForm = (): JSX.Element => {
+  const dispatch = useAppDispatch();
+
   const onSubmit: SubmitHandler<IRegisterData> = (data: IRegisterData) => {
-    console.log(data);
+    if (data.email && data.password && data.firstName && data.dateOfBirth && data.country) {
+      const year = data.dateOfBirth.getFullYear();
+      const month = String(data.dateOfBirth.getMonth() + 1).padStart(2, '0');
+      const day = String(data.dateOfBirth.getDate()).padStart(2, '0');
+      const formattedDate = `${year}-${month}-${day}`;
+      const requestData = {
+        email: data.email,
+        firstName: data.firstName,
+        lastName: data.lastName,
+        password: data.password,
+        dateOfBirth: formattedDate,
+        addresses: [
+          {
+            firstName: data.firstName.trim(),
+            lastName: data.lastName.trim(),
+            streetName: data.street.trim(),
+            postalCode: data.postalCode,
+            city: data.city,
+            country: data.country.slice(0, 2).toUpperCase(),
```

Comment:
Here will be mistake. I will fix it in my next future. This schema is not suitable for  every country